packages/@uppy/companion/src/server/controllers/callback.js

```diff
@@ -27,5 +39,6 @@ module.exports = function callback (req, res, next) { // eslint-disable-line no-
 
   logger.debug(`Did not receive access token for provider ${providerName}`, null, req.id)
   logger.debug(grant.response, 'callback.oauth.resp', req.id)
-  return res.sendStatus(400)
+
```

Comment:
Should we keep the status code to `400`? It feels wrong to send `200` when there's clearly an error.
```suggestion
res.statusCode = 400
```

Reply:
I'm not sure whether some browsers will show a different page or error document if the code is not 400, do you know?

Reply:
Browsers render the HTML file whether the error code is 200 or something else (e.g.: https://uppy.io/page_that_does_not_exist), with the exception of 3xx error code that redirect the user before the page is displayed. There might be other exceptions that I don't know, but I don't think 400 is one of them.

Reply:
I have some faint memory that Internet explorer shows a custom error page when it receives an error http status code, but I may be wrong, and I'm not sure if we support IE any more